### PR TITLE
omit solr_bbox and stale metadata

### PR DESCRIPTION
**Pull request by drh-stanford**
see https://github.com/sul-dlss/earthworks/issues/199

also removes selected.json and solr_wxs_url which was deprecated long ago.

### CODE REVIEW

**Comment by drh-stanford** [comment]
depends on https://github.com/geoblacklight/geoblacklight/issues/297

**Comment by drh-stanford** [comment]
I pulled out the `selected.json` removal into a new PR https://github.com/sul-dlss/earthworks/pull/201

**Comment by mejackreed** [comment]
Looks like our tests are missing some things here. Removing the `solr_bbox` field breaks all spatial search. We also need to change the configuration field, plus pull in the upstream updates from GeoBlacklight.

**Comment by drh-stanford** [comment]
yes this pull request depends on geoblacklight/geoblacklight#297 so we need to upgrade whenever the next geoblacklight release is cut.

**Comment by mejackreed** [comment]
This can be rebased and merged once #209 is merged

**Comment by drh-stanford** [comment]
rebased

File: spec/fixtures/solr_documents/public_polygon_mit.json [hunk 1]
```diff
@@ -36,12 +36,7 @@
...
   "layer_modified_dt": "2014-05-27T18:09:34Z",
-  "solr_bbox": "-73.533237 41.230345 -69.898565 42.888068",
-  "solr_ne_pt": "42.888068,-69.898565",
-  "solr_sw_pt": "41.230345,-73.533237",
   "solr_geom": "ENVELOPE(-73.533237, -69.898565, 42.888068, 41.230345)",
   "solr_year_i": 2000,
-  "solr_issued_dt": "2000-01-01T00:00:00Z",
-  "solr_wms_url": "http://arrowsmith.mit.edu:8080/geoserver/wms",
-  "solr_wfs_url": "http://arrowsmith.mit.edu:8080/geoserver/wfs"
+  "solr_issued_dt": "2000-01-01T00:00:00Z"
```
Comment (mejackreed) [inline spec/fixtures/solr_documents/public_polygon_mit.json]: Looks like this snuck in @drh-stanford

**Comment by drh-stanford** [comment]
These fixtures are all in GeoBlacklight v1.0 now...

**Pull request closed without merging**